monai/data/image_reader.py

```diff
@@ -766,7 +766,11 @@ def _get_seg_data(self, img):
             all_segs = np.zeros([*spatial_shape, n_classes])
 
         for i, (frames, description) in enumerate(self._get_frame_data(img)):
-            class_name = description.SegmentDescription
+            class_name = (
+                description.SegmentDescription
```

Comment:
Could this be done as 'getattr(description, "SegmentDescription", description.SegmentLabel)`? This also assumes the `SegmentLabel` property is always present.